specs/splitbutton/splitbutton.spec.md

```diff
@@ -0,0 +1,220 @@
+  - Tab out of menu to dismiss
+
+Keyboard operation is very similar to the [WAI-ARIA example of MenuButton](https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-actions-active-descendant.html)
+
+### Risks and Challenges
+
+- It depends on fast-button and fast-popover.
+
+- Because the  collection of related action buttons are displayed in a separate popover/flyout/listbox, a developer must know whether and how to use a stateful split button or standard split button and all the extra properties needed for the stateful one. 
+
+Much of the interaction is handled not by the base split button, but by the popover/flyout/menu:
+  - Should the menu of buttons should be in the DOM once, no matter how many split buttons are in the page?
+
+Accessibility - it is important that the split button is two tab stops: one for the default button and one for the related actions in its menu.
+
+### Prior Art/Examples
+
+Split buttons:
+- [Fluent UI](https://developer.microsoft.com/en-us/fluentui#/controls/web/button)
+- [Bootstrap](https://getbootstrap.com/docs/4.3/components/dropdowns/#split-button)
+- [Microsoft UWP](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/buttons#create-a-split-button)
+- [Semantic UI](https://semantic-ui.com/modules/dropdown.html#floating)
+- [Telerik]( https://demos.telerik.com/aspnet-ajax/button/examples/splitbutton/defaultcs.aspx)
+
+Similar:
+- [Salesforce Lightning - button group with overflow](https://www.lightningdesignsystem.com/components/button-groups/#With-Overflow-Menu-2)
+- [Material Design - exposed dropdown](https://material.io/components/menus#exposed-dropdown-menu)
+- [w3c - menu action button](https://w3c.github.io/aria-practices/examples/menu-button/menu-button-actions-active-descendant.html)
+
+---
+
+## Design
+
+### API
+
+*Component Name:*
+
+- `fast-split-button`
+
+*Properties:*
+
+- `relatedButtons` - collection of buttons to be displayed in the popover
+- `relatedPopoverMenu` - a fast-popover used in the slotted directive for the flyout and related secondary actions
+- `stateful` - boolean. False (default) when it is a Fluent-style split button. True it is "like"-style split button
+- `has-divider` - boolean. True (default) shows the divider between primary-action-button and the menu-button. False if there is no divider, like the Teams example above.
+- `disabled` - boolean. Disables entire split button
+- `primary-disabled` - boolean. Disables just the primary action button, but leaves menu-button and menuitems enabled. For Fluent parity
+- `aria-haspopup` - true
+- `aria-controls` - reference to popover
+- `aria-expanded` - true when the popover is open
+
+*Slots:*
+- `primary-button` - the primary action button
+- `expand-glyph` - glyph that indicates the select is expandable, often a downward chevron
+- `related-buttons` - collection of buttons that will be displayed in the popover menu
```

Comment:
Will the pop-over only ever display buttons?

Reply:
Great question. It seems like it should only ever be buttons in a split button pop-over, but I could be wrong. Maybe a divider?

Reply:
Thanks Brook. A scenario comes to mind when an engineer might want to put something else in the popover. For instance, Github has many "split button" patterns that go from simple content:
![image](https://user-images.githubusercontent.com/5091822/114736035-9c1f8300-9cfa-11eb-911f-ec04d03e34bb.png)

To more complex content:
![image](https://user-images.githubusercontent.com/5091822/114736595-118b5380-9cfb-11eb-875e-1320cf6e5911.png)

This might be something we want to consider in this component.